CMakeLists.txt

```diff
@@ -3084,6 +3096,7 @@ if (gRPC_INSTALL)
   )
 endif()
 
+if (_gRPC_BUILD_CODEGEN)
```

Comment:
And I believe this is wrong (and all other instances of `_gRPC_BUILD_CODEGEN`), as it is `gRPC_BUILD_CODEGEN`.

Interestingly, the Android build passes even with the typo, but not the iOS one.

Reply:
Thanks for the catch! I suspect a non-defined option defaults to false, which makes Android happy here but will break the regular builds